### DISCUSSION TITLE
Ubuntu 21.10

### DISCUSSION THREAD

**Discussion by ginoconti** [Security-Onion-Solutions/securityonion#7429] [General] [answered]
Any word when SO2 will support the 21.10 version of ubuntu?

**Comment by dougburks** [comment, marked as answer]
We have no plans to support Ubuntu 21.10 since it is not an LTS version. 

We support Ubuntu 18.04 LTS and Ubuntu 20.04 LTS:
https://docs.securityonion.net/en/2.3/installation.html#installation-on-ubuntu-or-centos
